modules/custom/social_language/modules/social_content_translation/social_content_translation.install

```diff
@@ -5,72 +5,62 @@
  * Installation tasks for the Social Language Content Translation module.
  */
 
+use Drupal\Core\Config\FileStorage;
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\language\Entity\ContentLanguageSettings;
+
 /**
  * Implements hook_install().
  */
 function social_content_translation_install() {
-  social_content_translation_disable_field_translations();
+  social_content_translation_install_language_settings_for_entities();
 }
 
 /**
- * Disable field translations.
- *
- * Disable content translation for all fields in nodes, paragraphs and
- * taxonomy terms. They are enabled by default when turning on content
- * translation but we want to control this to only the fields that are in
- * ContentTranslationDefaultsConfigOverride classes.
+ * Implements hook_uninstall().
  */
-function social_content_translation_disable_field_translations() {
-  $entity_types = [
-    'node',
-    'paragraph',
```

Comment:
We have now only added this for topics, and removed all these, shouldn't we add more of these `language.content_settings.node.xxx.yml` files?

Reply:
@ronaldtebrake Actually, we tried to add these configs during working on translations compatibility in OS. So, probably they should already exist (if we don't miss something).
Of course, if we add a new content type we need to add this config as well.